src/wasm-binary.h

```diff
@@ -155,7 +155,8 @@ enum Section {
   Functions = 2,
   DataSegments = 4,
   FunctionTable = 5,
-  End = 6
+  End = 6,
+  WLLSegment = 0x11
```

Comment:
That's ok, I give up on this for now. It's probably better for it to completely fail using the binaryen tools. People will then be directed to WLL compatible tools, and unsuspecting developers will be less likely to unintentionally use the WLL section for other purposes. It seems a stopgap solution anyway. The WLL section could alternatively go after the end-section, along with the rest of the ool blobs, or have a signature at the end of the file - lots of possibilities.